demos/storybook/stories/drawer/with-nested-list-items.tsx

```diff
@@ -25,26 +25,53 @@ const componentLibrary = 'Component Library';
 const typographyRules = 'Typography Rules';
 const themeRules = 'Theme Rules';
 
+const getIcon = (icon: string): JSX.Element | undefined => {
+    switch (icon) {
+        case '<Add />':
+            return <Add />;
+        case '<PinDrop />':
+            return <PinDrop />;
+        case '<Remove />':
+            return <Remove />;
+        case '<AddAPhoto />':
+            return <AddAPhoto />;
+        case 'undefined':
+        default:
+            return undefined;
+    }
```

Comment:
This function is defined in several places, should we make it a utility?

Reply:
I tried, and then storybook tried to recognize every .tsx file as a story. I did not look deeper into that.